www/src/pages/technical-reports.md

```diff
@@ -6,9 +6,10 @@ layout: ../layouts/Markdown.astro
 
 These are the technical reports produced by the Design Tokens Community Group.
 
-| Name                                               | Published  |
-| :------------------------------------------------- | :--------- |
-| [Preview draft (unpublished)](/TR/drafts/)         | —          |
-| [Third Editors' Draft](/TR/third-editors-draft/)   | 2025-07-21 |
-| [Second Editors' Draft](/TR/second-editors-draft/) | 2022-06-14 |
-| [First Editors' Draft](/TR/first-editors-draft/)   | 2021-09-23 |
+| Name                                               | Status | Published  |
+| :------------------------------------------------- | :----- | :--------- |
+| [Preview draft (unpublished)](/TR/drafts/)         |        | —          |
+| [2025.10](/TR/2025.10/)                            | Stable | 2025-10.27 |
```

Comment:
Anyone have thoughts on “Status” as a column? I wanted to show some distinction between the drafts and non-drafts. Should the other ones be “draft”?

Reply:
[Kaelig] No this is great, we can iterate if needed but it's a great improvement that helps differentiate between versions at a glance.